### PR TITLE
APP-23022: Define the static variable `__DEV__`

### PR DESCRIPTION
**Pull request by alexisfontaine**
Automatically created JIRA issue: [APP-23022](https://habxfr.atlassian.net/browse/APP-23022)

### CODE REVIEW

File: webpack/overrides.js [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+const webpack = require('webpack')
 
 const helpers = require('../helpers')
 const customConfig = require('./webpack.config')
 
 module.exports = (defaultConfig) => {
   let config = merge({}, defaultConfig, customConfig)
 
+  config.plugins.unshift(new webpack.DefinePlugin({
+    __DEV__: JSON.stringify(process.env.NODE_ENV !== "production"),
```
Comment (jean9696) [inline webpack/overrides.js:11]: Why do you need to stringify a Boolean ?

Comment (alexisfontaine) [reply webpack/overrides.js:11]: Because I [copy-pasted](https://github.com/apollographql/react-apollo-error-template/pull/51) it :sweat_smile:

Comment (alexisfontaine) [reply webpack/overrides.js:11]: Hint: I never use double quotes.

Comment (jean9696) [reply webpack/overrides.js:11]: But this results in a string instead of boolean 🤔 Weird...

**Review by jean9696 — APPROVED** [review]

**Comment by benjamn** [comment]
Drive-by comment: I misinterpreted https://webpack.js.org/plugins/define-plugin/#usage as suggesting that `JSON.stringify` was the way to include that string's value as code, but (reading the rest of the rules) it turns out `JSON.stringify` is only necessary when the value is already a string (to preserve the quotation marks). Other values will be stringified automatically, and `true` and `false` have no trouble with that. Simplified in https://github.com/apollographql/react-apollo-error-template/pull/57.

**Pull request merged**